pkg/volume/testing/testing.go

```diff
@@ -248,6 +250,49 @@ func (f *fakeVolumeHost) GetEventRecorder() record.EventRecorder {
 	return nil
 }
 
+func (f *fakeVolumeHost) ScriptCommands(scripts []CommandScript) {
+	ScriptCommands(f.exec, scripts)
+}
+
+type CommandScript struct {
```

Comment:
Can you add some comments to this struct about what each of the fields do?

Reply:
Done.